be1-go/hub/organizer/mod.go

```diff
@@ -291,6 +295,30 @@ func (h *Hub) handleMessageFromClient(incomingMessage *socket.IncomingMessage) e
 	return nil
 }
 
+func verifySignature(publicKey64 string, signature64 string, data64 string) error {
+	pk, err := base64.URLEncoding.DecodeString(publicKey64)
+	if err != nil {
+		return xerrors.Errorf("failed to decode the public key")
```

Comment:
It could be a good idea to return err too, and same comment for the returns below

Reply:
nice catch!